packages/client-twitter/src/post.ts

```diff
@@ -30,21 +35,42 @@ const twitterPostTemplate = `
 Write a 1-3 sentence post that is {{adjective}} about {{topic}} (without mentioning {{topic}} directly), from the perspective of {{agentName}}. Do not add commentary or acknowledge this request, just write the post.
 Your response should not contain any questions. Brief, concise statements only. The total character count MUST be less than {{maxTweetLength}}. No emojis. Use \\n\\n (double spaces) between statements.`;
 
+export const twitterActionTemplate = `
+# INSTRUCTIONS: Determine actions for {{agentName}} (@{{twitterUserName}}) based on:
+{{bio}}
+{{postDirections}}
+
+Guidelines:
+- Highly selective engagement
+- Direct mentions are priority
+- Skip: low-effort content, off-topic, repetitive
+
+Actions (respond only with tags):
+[LIKE] - Resonates with interests (9.5/10)
+[RETWEET] - Perfect character alignment (9/10)
+[QUOTE] - Can add unique value (8/10)
+[REPLY] - Memetic opportunity (9/10)
+
+Tweet:
+{{currentTweet}}
+
+# Respond with qualifying action tags only.`
+    + postActionResponseFooter;
+
+const MAX_TWEET_LENGTH = 240;
+
 /**
  * Truncate text to fit within the Twitter character limit, ensuring it ends at a complete sentence.
  */
-function truncateToCompleteSentence(
-    text: string,
-    maxTweetLength: number
```

Comment:
not sure I like this being removed, it's a better function with this versatility